### ISSUE TITLE
Include zen of prometheus

### ISSUE THREAD

**Issue by paulfantom** [prometheus/docs#1692]
Include @kakkoyun "zen of prometheus" site (or its variation) under prometheus.io/zen.

I believe it would be a useful addition to documentation to include a set of best practices in a catchy form. Values defined by @kakkoyun in https://the-zen-of-prometheus.netlify.app/ would be great way to do this.

cc: @SuperQ

**Comment by kakkoyun** [comment]
This is a great idea! I'm gonna send a PR to add this page. Let's see what maintainers would say.

**Comment by brian-brazil** [comment]
Linking from the media page is probably the best thing.

**Comment by RichiH** [comment]
I think either linking to it or moving it into our official docs are the two feasible options; having two reference implementations will mean divergence and toil in the long run.

@kakkoyun, which of the two would you prefer?

**Comment by kakkoyun** [comment]
@RichiH @brian-brazil I would love to move it to the official documents, make it more visible and a community maintained document.

**Comment by RichiH** [comment]
I have an age-old branch for a philophy section somewhere. This would be a good fit.

**Comment by kakkoyun** [comment]
@RichiH Will that branch of yours become a PR? Do you have any pointers for that? 
Otherwise which section I should put the zen under? Best practices?

**Comment by brian-brazil** [comment]
Best practices would be it, but if it's on the site we need to be careful that pithiness doesn't lead to confusion.

**Comment by kakkoyun** [comment]
Yes, of course. We can iterate on the PR to prevent any misguiding.
